templates/vue/src/components/Lightbox/media/MultiContentMedia/PageRows.vue

```diff
@@ -47,10 +47,7 @@
                 @complete="updateProgress(row.node.id)"
                 @load="handleLoad($refs.rowRefs[index])"
               />
-              <p
-                v-if="row.children.length > 0 && !areAllPopup(row.children)"
-                style="color: white;"
-              >
+              <p v-if="row.children.length > 0" style="color: white;">

```

Comment:
How come you have removed the `areAllPopup` check here?

Reply:
Woops, my mistake. Added it back in.